### PR TITLE
Dont error on returned constant, fix for levit_128

### PR DESCRIPTION
**Pull request by eellison**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #113389
* __->__ #112544
* #112543

Previously, levit_128 would fail on inference because we would return a view of a constant, which messed up our assertions of outputs being in the cuda graph pool.

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov @ColinPeppler

### CODE REVIEW

File: torch/_inductor/cudagraph_trees.py [hunk 1]
```diff
@@ -582,14 +585,18 @@ def run(self, new_inputs):
...
+
+        def get_non_cudagraph_inps():
+            non_cudagraph_inps = set()
+            for t in itertools.chain(new_inputs, self.wrapped_function.constants):
+                if (
+                    isinstance(t, torch.Tensor)
+                    and t.untyped_storage().data_ptr() not in existing_path_data_ptrs
+                ):
+                    non_cudagraph_inps.add(t.untyped_storage().data_ptr())
+            return non_cudagraph_inps
```
Comment (ezyang) [inline torch/_inductor/cudagraph_trees.py:597]: poor man's scoping?

Comment (eellison) [reply torch/_inductor/cudagraph_trees.py:597]: previously, i had this inlined and at the end `del t` but `t` is not always set... better suggestion ?

**Review by ezyang — APPROVED** [review]
I don't really understand but this seems plausible

**Pull request closed without merging**